tests/fixtures/fixture_model.py

```diff
@@ -364,3 +374,88 @@ def coupled_mf6_model_storage_coefficient() -> mf6.Modflow6Simulation:
     coupled_mf6_model["GWF_1"] = gwf_model
 
     return coupled_mf6_model
+
+
+@pytest_cases.fixture(scope="session")
+def prepared_dflowfm_model(dflowfm_initial_inputfiles_folder) -> FMModel:
+    x, y, _, times, dx, dy, _ = grid_sizes()
+
+    # create a temp directory and make it the working directory. Store the current working directory
+    # so that we can go back to it on leaving the function 
+    currdir = getcwd()
+    workdir = tempfile.mkdtemp()
```

Comment:
You can use a fixture for that, see e.g. here: https://github.com/Deltares/imod_coupler/blob/6fd4928fc0648deb0952f3d5cca89b4a8326052a/tests/fixtures/fixture_paths.py#L10

Reply:
done